source/extensions/filters/http/grpc_field_extraction/filter_config.cc

```diff
@@ -0,0 +1,89 @@
+#include "source/extensions/filters/http/grpc_field_extraction/filter_config.h"
+
+#include "source/common/common/fmt.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace GrpcFieldExtraction {
+
+FilterConfig::FilterConfig(
+    const envoy::extensions::filters::http::grpc_field_extraction::v3::GrpcFieldExtractionConfig&
+        proto_config,
+    std::unique_ptr<ExtractorFactory> extractor_factory, Api::Api& api)
+    : proto_config_(proto_config) {
+  type_helper_ = std::make_unique<google::grpc::transcoding::TypeHelper>(
+      Protobuf::util::NewTypeResolverForDescriptorPool(Envoy::Grpc::Common::typeUrlPrefix(),
+                                                       &descriptor_pool_));
+
+  initDescriptorPool(api);
+  initExtractors(*extractor_factory);
+}
+
+void FilterConfig::initExtractors(ExtractorFactory& extractor_factory) {
+  for (const auto& it : proto_config_.extractions_by_method()) {
+    auto* method = descriptor_pool_.FindMethodByName(it.first);
+    if (method == nullptr) {
+      throw EnvoyException(fmt::format(
+          "couldn't find the gRPC method `{}` defined in the proto descriptor", it.first));
```

Comment:
nit: provide some suggestions on how to fix config

For example, remind user of the syntax to specify gRPC method in filter config. Or have them verify gRPC descriptor was updated

Reply:
IMHO, it may be unneeded as the proto descriptor and the filter config use the proto path style.